appinfo/routes.php

```diff
@@ -32,6 +32,10 @@
 				'url' => '/accounts/{accountId}/folders/{folderId}/messages/{messageId}/toggleStar',
 				'verb' => 'POST'],
 			[
+				'name' => 'messages#detectChanges',
```

Comment:
> why is there another route? I don't get this - sorry

Will be merged with the existing ``folders#detectChanges`` route, I added this just to try things out ;-)